type.c

```diff
@@ -677,97 +739,36 @@ int MPI_Type_ub(MPI_Datatype type, MPI_Aint * ub)
   return Type_ub(type_ptr, ub);
 }
 
-int Type_ub(Datatype type, MPI_Aint * ub)
-{
-  *ub = type->ub;
-}
-
 /* MPI_Get_address
  * MPI_Address
  * Return address of an object
  */
-FC_FUNC( mpi_get_address, MPI_ADDRESS )(void * loc, long * address, int * ierr)
+int FGet_address(void * loc, long * address)
```

Comment:
Note the deletion of the ierr argument from this function. clang flagged that as an error, and it looked like a real error to me, but someone should double-check that.